packages/uploadthing/src/internal/logger.ts

```diff
@@ -73,15 +73,7 @@ function formatArgs(args: any[]) {
     if (typeof arg === "string") {
       return arg;
     }
-    try {
-      // prefer inspect over JSON.stringify because it handles circular references, prints classes etc
-      // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/consistent-type-imports
-      const util = require("util") as typeof import("util");
-      return util.inspect(arg, { depth: 4 });
-    } catch {
-      // fallback to JSON.stringify if inspect fails e.g. if runtime doesn't have util module
-      return JSON.stringify(arg, null, 4);
-    }
+    return JSON.stringify(arg, null, 4);
```

Comment:
downside is if we attempt to log classes `Response` it will just output an empty object, but we don't do that anywhere today and according to my tests, logs looks good with JSON.stringify:

![CleanShot 2024-03-01 at 21 21 12@2x](https://github.com/pingdotgg/uploadthing/assets/51714798/b507087e-8a3c-4ad5-b0dc-cabb2eb4019c)